cmd/ctr/run_unix.go

```diff
@@ -84,9 +84,9 @@ func newContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 	)
 	cOpts = append(cOpts, containerd.WithContainerLabels(labelArgs(context.StringSlice("label"))))
 	if context.Bool("rootfs") {
-		opts = append(opts, containerd.WithRootFSPath(ref, context.Bool("readonly")))
+		opts = append(opts, containerd.WithRootFSPath(imageNameOrPath, context.Bool("readonly")))
 	} else {
-		image, err := client.GetImage(ctx, ref)
+		image, err := client.GetImage(ctx, imageNameOrPath)
```

Comment:
This method does not take a path. Do we use that as the name in this case?

Reply:
Path when --rootfs is set